src/tests/webkit/WebKit_Performance_001.test.js

```diff
@@ -1,35 +1,59 @@
-import { setWebKitUrl, calcAvgFPS, webKitBrowserActions } from '../../commonMethods/commonFunctions'
+import {
+  setWebKitUrl,
+  calcAvgFPS,
+  webKitBrowserActions,
+  pluginActivate,
+  suspendOrResumePlugin,
+  pluginDeactivate,
+} from '../../commonMethods/commonFunctions'
 import { fetchWebKitFPS } from '../../commonMethods/webKitPerformanceCommonFunctions'
 import constants from '../../commonMethods/constants'
 
 let listener
 
 export default {
   context: {
-    minFPS: 40,
-    url: 'http://webkit.org/blog-files/3d-transforms/poster-circle.html',
+    minFPS: 30,
+    url: 'https://webkit.org/blog-files/3d-transforms/poster-circle.html',
   },
   setup() {
     this.$data.write('samples', [])
     return this.$sequence([
-      () => webKitBrowserActions.call(this),
-      () =>
-        (listener = this.$thunder.api.WebKitBrowser.on('urlchange', data => {
-          this.$data.write('currentUrl', data.url)
-        })),
+      () => pluginDeactivate.call(this, 'WebKitBrowser'), //cycle the browser
```

Comment:
Instead of "//cycle the browser", we can write '//make sure Webkit Browser is turned off'

Reply:
sure!